Source/Scene/SkyAtmosphere.js

```diff
@@ -76,6 +76,11 @@ define([
         this._command = new DrawCommand({
             owner : this
         });
+        this._defaultColor = {
+          red: 1,
+          green: 1,
+          blue: 1
+        };
```

Comment:
Cesium has a [Color class](http://cesiumjs.org/Cesium/Build/Documentation/Color.html), please use that instead of this data structure.